critiquebrainz/frontend/templates/moderators/moderators.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ _('Moderators') }} - CritiqueBrainz{% endblock %}
+
+{% block content %}
+<div class="container">
+  <h1>{{ _('Moderators') }}</h1>
+  <div id="list" style="margin-top: 30px">
```

Comment:
Few lines here, few lines there, and it becomes a mess. We already have a place where custom styling for pages must go: https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/frontend/static/styles/main.less.